examples/cool_features_demo.py

```
@@ -0,0 +1,319 @@
+    
+    conn.close()
+    print()
+
+
+def demo_builder_with_validation():
+    """Demonstrate builder with validation errors."""
+    print("=== 🚨 Builder with Validation Errors ===")
+    
+    try:
+        # This should fail validation
+        person = (Person.builder()
```

Ruff (0.12.2)

Local variable `person` is assigned to but never used

Remove assignment to unused variable `person`

(F841)